src/components/custom/VoteResultsSection.js

```diff
@@ -0,0 +1,40 @@
+import {VotingNotEndedCard} from "@/components/custom/VotingNotEndedCard";
+import {ResultsCard} from "@/components/custom/ResultsCard";
+import {VotingForm} from "@/components/custom/VotingForm";
+import {useState} from "react";
+import {useStore} from "@/lib/store";
+
+export function VoteResultsSection({project, change, changeVotes, userVoteProp}) {
+	const [userRoles, setInvalidPermissionsDialogOpen] = useStore((state) =>
+		[state.userRoles, state.setInvalidPermissionsDialogOpen]
+	);
+	const [userVote, setUserVote] = useState(userVoteProp);
+
+	const userCanVote = () => {
+		if (!userRoles.find((role) => role.project === project.pid && role.role.vote_on_change)) {
+			setInvalidPermissionsDialogOpen(true);
+			return false;
+		}
+		return true;
+	}
+
+	return (
+		<>
+			{(change.closed || change.merged) ? (
+				<ResultsCard change={change} changeVotes={changeVotes}/>
+			) : (
+				userCanVote() ? (
+					<>
+						<VotingForm change={change} userVote={userVote} setUserVote={setUserVote}/>
+						<VotingNotEndedCard/>
+					</>
+				) : (
+					<>
+						<h2 className="mt-4 text-lg">Results</h2>
```

Comment:
Shouldn't this be part of the `VotingNotEndedCard`? So that the heading of `Results` always shows. That's how it is in the Figma

Reply:
fixed.